### DISCUSSION TITLE
Specification for regex

### DISCUSSION THREAD

**Discussion by HaleTom** [espanso/espanso#1884] [Q&A] [answered]
The [Regex triggers doco](https://espanso.org/docs/matches/regex-triggers/) doesn't say what implementation of regex is used.

Could you please provide a link to the spec so we can regex all the way up to the guard rails?

**Comment by smeech** [comment, marked as answer]
The [docs](https://espanso.org/docs/matches/regex-triggers/#arguments) refer to https://docs.rs/regex/1.5.4/regex/, so I have my https://regex101.com/ default set to Rust, which has worked for me up to now.
